UKHomeOffice/sgmr-service
# NMSW-504 Change 'gender' to 'sex' and 'non-binary' to 'other'

## AC
Change "Gender" to "Sex"
Change "non-binary" to "Other"

## TO TEST

- go to people page
- add a new person
- > field should as for Sex not Gender
- > radio buttons should be male, female, other
- click continue
- > error should show Sex not Gender
- create a person with a sex of other
- click on update to return to that person
- > other should be populated
